core/trino-spi/pom.xml

```diff
@@ -254,6 +254,77 @@
                                     <old>interface io.trino.spi.ptf.TableFunctionDataProcessor</old>
                                     <justification>Relocate table function infrastructure</justification>
                                 </item>
+                                <item>
```

Comment:
Add a commit before this to delete all the previous exclusions, since they're for the previous release